### PR TITLE
Added a minimal language field parser

### LINKED ISSUE #23: How about a language field in the seri description ?
Since not everyone is comfortable about giving a talk of english, should we add a field to the `seri` language to define the language of a talk ? Similarly to the *speakers* attribute, the *lang* attribute would be useful only for talks.

Any opinion on this ?

### PR DESCRIPTION
**Pull request by ororktn**
Still need to add it to other backends, edit it for abstracts backend, rustfmt, and clippy

### CODE REVIEW

File: data/template_abstex.tex [hunk 1]
```diff
@@ -1,5 +1,10 @@
 \documentclass[a4paper]{article}
 
+\usepackage[T1]{fontenc}
+\usepackage[utf8]{inputenc}
+
+\usepackage{emoji}
+
```
Comment (Lymkwi) [inline data/template_abstex.tex:19]: Considering these changes will only work with LuaLaTeX, should we change the README to signal it? Or change the code to handle regular non UTF-8 compilers?

File: src/event.rs [hunk 2]
```diff
@@ -68,6 +70,8 @@ pub struct Event {
     pub duration: u32,
     /// The event description
     pub description: Option<String>,
+    /// Potentially the language of the talk
```
Comment (hreymond) [inline src/event.rs]: Enlever le potentially ?

File: src/lang.rs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    fn from_str(input: &str) -> Result<Self, Self::Err> {
+        match input.to_lowercase().as_str() {
+            "fr" => Ok(Self::French),
+            "en" => Ok(Self::English),
+            tt => Err(InvalidLanguage(tt.to_owned())),
+        }
+    }
+}
+
+impl fmt::Display for Lang {
```
Comment (Lymkwi) [inline src/lang.rs]: Maybe use `ToString` so it auto derives `Display`?

File: src/event.rs [hunk 4]
```diff
@@ -148,6 +152,12 @@ impl FromStr for Event {
                 Type::from_str(talk_type).map_err(ParsingError::from)
             })?;
 
+        let language = if let Some(lang) = settings.get("lang").as_ref() {
```
Comment (Lymkwi) [inline src/event.rs]: `as_ref` is useless here because the language implements `Copy`

**Review by Lymkwi — COMMENTED** [review]
Good, see the comments!

File: src/lang.rs (hunk 3 above)
Comment (Lymkwi) [reply src/lang.rs]: My bad, the opposite is recommended.

File: src/event.rs [hunk 5]
```diff
@@ -148,6 +152,12 @@ impl FromStr for Event {
...
+            Lang::from_str(lang).ok()
+        } else {
+            None
+        };
+
```
Comment (hreymond) [inline src/event.rs:155]: Maybe use `map_or` to improve readability

```rust
let language = settings
            .get("lang")
            .map_or(None, |l| Lang::from_str(l).ok());
```

**Review by hreymond — COMMENTED** [review]
Thanks for this addition ! 
I just have minors comment, cf review

**Pull request merged**